File: MasterErp.Angular/src/app/components/GeneralAccounts/models/GeneralAccounts/DailyNotebookModel.ts
import { CreatorModel } from "src/app/components/Shared/models/CreatorModel";

export interface DailyNotebookModel extends CreatorModel {
    dailyNotebookId: number | null;
    nameEN: string;
    nameAR: string;
    ledgerTypeId: number;
    code: string;
    virtualAccount: string;
    receiptLedgerTypeNameEN: string;
    receiptLedgerTypeNameAR: string;
    totalCount: number | null;
}

File: MasterErp.Entities/Common/Finance/GeneralAccounts/DailyNotebookModel.cs
using MasterErp.Entities.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MasterErp.Entities.Common.Finance.GeneralAccounts
{

    public class DailyNotebookModel : CreatorModel
    {
        public int? DailyNotebookId { get; set; }
        public string NameEN { get; set; }
        public string NameAR { get; set; }
        public int LedgerTypeId { get; set; }
        public string Code { get; set; }
        public string VirtualAccount { get; set; }
        public string ReceiptLedgerTypeNameEN { get; set; }
        public string ReceiptLedgerTypeNameAR { get; set; }
        public int? TotalCount { get; set; }

    }
}
